content/basics/basics.mdx

```diff
@@ -0,0 +1,327 @@
+---
+name: Basics
+route: /basics
+---
+
+import { Playground } from 'docz'
+
+# Basic Introduction
+
+[MobX](https://mobx.js.org/README.html) lets you create observable state. This
+library, `mobx-react` integrates it with React. It provides a way for you to
+create React components that respond to MobX state changes.
+
+> Frameworks such as [mobx-state-tree](https://mobx-state-tree.js.org) and
+> [mobx-keystone](https://mobx-keystone.js.org/) build on
+> top of MobX to offer a higher-level way to create observable models. These models
+> give you additional properties, such as automatic serialization to JSON. This
+> introduction uses plain MobX models. The principles of React
+> integration are the same for these advanced libraries are the same
+> as they are also built with MobX.
+
+## MobX models
+
+Let's consider MobX models for animals in a zoo, where each animal
+has a certain amount of energy. If the energy is below 50, the animal
+is hungry. Animals can be fed to increase their energy levels:
+
+> These code examples use TypeScript. If you use plain JavaScript you
+> can skip the type declarations.
+
+```tsx
+import { observable, action, computed } from 'mobx'
+
+class Animal {
+  name: string
+  species: string
+  @observable energy: number
+
+  constructor(name: string, species: string, energy: number = 100) {
+    this.name = name
+    this.species = species
+    this.energy = energy
+  }
+
+  @action
+  timePasses() {
+    this.energy -= 10
+  }
+
+  @action
+  feed() {
+    this.energy += 50
+    if (this.energy > 100) {
+      this.energy = 100
+    }
+  }
+
+  @computed
+  get isHungry() {
+    return this.energy < 50
+  }
+}
+
+class Zoo {
+  @observable animals: Animal[] = []
+
+  @action
+  addAnimal(animal: Animal) {
+    this.animals.push(animal)
+  }
+}
+```
+
+## A React component that observes
+
+We haven't used anything except plain MobX in this example. Now let's
+create a React component to show information for an animal:
+
+```tsx
+import { Component } from 'react'
+import { observer } from 'mobx-react'
+
+const AnimalView = observer(({ animal }: { animal: Animal }) => {
```

Comment:
It's a matter of preference, but I think this is more readable as there is less animals together :)
```suggestion
type TProps = {
  animal: Animal
}
const AnimalView = observer<TProps>(({ animal }) => {
```